### PR TITLE
fix: pagination issues

### LINKED ISSUE #138: Pagination counting bug when mirrors are added or deleted
### Describe the bug

Not sure of the cause here but the pagination value is off when mirrors are added / deleted from the list.

### To Reproduce

1. Pull up a list of mirrors
2. Delete a mirror
3. Note the count reflected on the pagination nav differs from the actual number of mirrors.

### Expected behavior

The navigation UI should match reality.

### Screenshots

_No response_

### Additional context

Image

### PR DESCRIPTION
**Pull request by sutterj**
# Pull Request

## Proposed Changes

- Closes #138
  - increase items per page
  - fix issue where body scrolled over header
  - fix issue with pagination count and search
  - due to limitations with primer table pagination, there is no way to fix the count when the 
    last item on a table page is deleted, so when we run into this situation, reload the page

### Author/Contributor

- [ ] If documentation is needed for this change, has that been included in this pull request
- [X] run `npm run lint` and fix any linting issues that have been introduced
- [X] run `npm run test` and run tests
- [ ] If publishing new data to the public (scorecards, security scan results, code quality results, live dashboards, etc.), please request review from `@jeffrey-luszcz`

### Reviewer

- [X] Label as either `bug`, `documentation`, `enhancement`, `infrastructure`, `maintenance`, or `breaking`

### CODE REVIEW

File: src/app/[organizationId]/forks/[forkId]/page.tsx [hunk 1]
```diff
@@ -318,7 +329,7 @@ const Fork = () => {
                 align: 'end',
               },
             ]}
-            rows={5}
+            rows={10}
```
Comment (zkoppert) [inline src/app/[organizationId]/forks/[forkId]/page.tsx:332]:
```suggestion
            rows={pageSize}
```

File: src/app/[organizationId]/page.tsx [hunk 2]
```diff
@@ -75,7 +75,7 @@ const Organization = () => {
                 width: 'auto',
               },
             ]}
-            rows={5}
+            rows={10}
```
Comment (zkoppert) [inline src/app/[organizationId]/page.tsx:78]:
```suggestion
            rows={pageSize}
```

File: src/app/page.tsx [hunk 3]
```diff
@@ -41,7 +41,7 @@ const Home = () => {
                 rowHeader: true,
               },
             ]}
-            rows={5}
+            rows={10}
```
Comment (zkoppert) [inline src/app/page.tsx:44]:
```suggestion
            rows={pageSize}
```

**Review by zkoppert — APPROVED** [review]
Code side looks good. Would be great to have another reviewer test this out in the app. I don't have a local dev env setup to see this running yet.

**Review by ajhenry — APPROVED** [review]
Works great on my end!

**Pull request merged**